go.mod

```diff
@@ -60,7 +60,7 @@ require (
 	go.opentelemetry.io/otel/sdk/metric v1.34.0
 	go.opentelemetry.io/otel/trace v1.34.0
 	go.temporal.io/api v1.48.1-0.20250426005355-d6c89de822db
-	go.temporal.io/sdk v1.34.0
+	go.temporal.io/sdk v1.34.1-0.20250429051454-eb7575730f9a
```

Comment:
Are we going to release this SDK version before the next server release? We don't want an untagged SDK version in a server release.

Reply:
Removed this change from this PR. But it's needed in the next PR.